src/service.h

```diff
@@ -51,10 +51,17 @@ class ServiceLayer final {
                          std::vector<std::string> &followed_by_user,
                          const std::string &username);
   // // Cindy's Implementation for streaming chirps with hashtags
-  std::vector<chirp::Chirp> stream(const std::string hashtag);
-  // This function is called in chirp() to determine if the text consist of an
-  // hashtag. If so, save that chirp into a proto Hashtags.
+  std::vector<chirp::Chirp> stream(const std::string hashtag,
+                                   std::time_t seconds,
+                                   int64_t microseconds_since_epoch);
+  // This function is called in chirp() to determine if the text consist of
+  // an hashtag. If so, save that chirp into a proto Hashtags.
   std::string CheckIfHaveHashtag(const std::string &text);
+  // This function is a helper function to find multiple hashtags in a chirp, if
+  // exists
+  std::vector<std::string> CheckIfHaveMultipleHashtags(const std::string &text);
```

Comment:
Should these be private helper functions?

Reply:
I wanted to test the function. I thought it's okay to put them here since service.h is like a middle layer class?